### PR TITLE
Update test adapter for dev17

### PR DESCRIPTION
**Pull request by spebl**

### CODE REVIEW

File: BoostTestPlugin/source.extension.vsixmanifest [hunk 1]
```diff
@@ -1,39 +1,39 @@
...
+        <MoreInfo>https://go.microsoft.com/fwlink/?linkid=853712</MoreInfo>
+        <License>use.txt</License>
+        <GettingStartedGuide>https://go.microsoft.com/fwlink/?linkid=853711</GettingStartedGuide>
+        <ReleaseNotes>Release Notes.txt</ReleaseNotes>
+        <Icon>Icons\Icon.ico</Icon>
+        <PreviewImage>Boost_preview.png</PreviewImage>
+        <Tags>C++, Unit Testing, Testing, test explorer, TDD, Boost, Boost Test Library, Visual Studio, Test Adapter</Tags>
+    </Metadata>
+    <Installation>
+        <InstallationTarget Version="[15.0,)" Id="Microsoft.VisualStudio.Pro" />
```
Comment (Codiferous) [inline BoostTestPlugin/source.extension.vsixmanifest:17]: Do these need to be updated to 17.0 or no?

File: BoostTestPlugin/source.extension.vsixmanifest [hunk 2]
```diff
@@ -1,39 +1,39 @@
...
+        <Asset Type="Microsoft.VisualStudio.Assembly" d:Source="Project" d:ProjectName="VisualStudioAdapter" Path="|VisualStudioAdapter|" AssemblyName="|VisualStudioAdapter;AssemblyName|" />
+        <Asset Type="Microsoft.VisualStudio.Assembly" d:Source="Project" d:ProjectName="Antlr.DOT" Path="|Antlr.DOT|" AssemblyName="|Antlr.DOT;AssemblyName|" />
+        <Asset Type="Microsoft.VisualStudio.Assembly" d:Source="Project" d:ProjectName="BoostTestShared" Path="|BoostTestShared|" AssemblyName="|BoostTestShared;AssemblyName|" />
+        <Asset Type="Microsoft.VisualStudio.VsPackage" d:Source="Project" d:ProjectName="BoostTestPackage" Path="|BoostTestPackage;PkgdefProjectOutputGroup|" />
+        <Asset Type="Microsoft.VisualStudio.ItemTemplate" d:Source="Project" d:ProjectName="BoostTestItemTemplate" d:TargetPath="|BoostTestItemTemplate;TemplateProjectOutputGroup|" Path="ItemTemplates" d:VsixSubPath="ItemTemplates" />
+    </Assets>
+    <Prerequisites>
+        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.8.27729.1,)" DisplayName="Visual Studio core editor" />
```
Comment (Codiferous) [inline BoostTestPlugin/source.extension.vsixmanifest:36]: These are also 15.8. I don't know if they should be 17.0 or not.

File: VisualStudioAdapter/VisualStudioAdapter.csproj [hunk 3]
```diff
@@ -1,92 +1,72 @@
...
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Import Project="$(NuGetPackages)\Microsoft.VisualStudioEng.MicroBuild.Core.0.4.1\build\Microsoft.VisualStudioEng.MicroBuild.Core.targets" Condition="Exists('$(NuGetPackages)\Microsoft.VisualStudioEng.MicroBuild.Core.0.4.1\build\Microsoft.VisualStudioEng.MicroBuild.Core.targets')"/>
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
-  </Target>
-  <Target Name="AfterBuild">
-  </Target>
-  -->
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (Codiferous) [inline VisualStudioAdapter/VisualStudioAdapter.csproj:1]: Surprisingly large diff here. I don't know if it signals anything strange.

**Review by Codiferous — APPROVED** [review]

File: VisualStudioAdapter/VisualStudioAdapter.csproj (hunk 3 above)
Comment (spebl) [reply VisualStudioAdapter/VisualStudioAdapter.csproj:1]: yeah noticed that too, probably just something with the line breaks, not too worried.

File: BoostTestPlugin/source.extension.vsixmanifest (hunk 1 above)
Comment (spebl) [reply BoostTestPlugin/source.extension.vsixmanifest:17]: Tested it locally and this seems to still work if installed into the older versions. It's not recommended and wont happen through the installer, but it looks like we don't need to fully limit it.

File: BoostTestPlugin/source.extension.vsixmanifest (hunk 2 above)
Comment (spebl) [reply BoostTestPlugin/source.extension.vsixmanifest:36]: Same as above, should be safe and seemed ok in testing.

**Pull request merged**